Cargo.toml

```diff
@@ -41,10 +26,25 @@ icon = "icons/app_near_40px.gif"
 [package.metadata.ledger.apex_p]
 icon = "icons/app_near_32px.png"
 
+[dependencies]
```

Comment:
@yogh333 this block of code changed to here, because i used `cargo bump minor`, let me know if there is any problem here.